src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -316,6 +316,14 @@ internal bool ServerMode
             }
         }
 
+        internal bool ShowVersion
+        {
+            get
+            {
+                return _showVersion;
```

Comment:
to match the current code style

---
In reply to: [100213197](https://github.com/PowerShell/PowerShell/pull/3115#discussion_r100213197) [](ancestors = 100213197)